### ISSUE TITLE
how can I find the data please

### ISSUE THREAD

**Issue by x-joey** [wangruohan1995/SpliceFinder#1]
`def load_data():

    labels = np.loadtxt('label.txt')
    encoded_seq = np.loadtxt('encoded_seq.txt')
    
    x_train,x_test,y_train,y_test = train_test_split(encoded_seq,labels,test_size=0.2)

    return np.array(x_train),np.array(y_train),np.array(x_test),np.array(y_test)`
when I read the file CNN_model.py,I don't know the format about 'encoded_seq.txt' data. could you give me some help?
best wish
